src/Filesystem/DTO/FileSize.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ILIAS\Filesystem\DTO;
```

Comment:
`DTO` is an abbreviation of data transfer object.
Thats just a habit of mine, I will rename the package to make it more explicit.